src/Sema.zig

```diff
@@ -35641,24 +35546,18 @@ fn resolveInferredErrorSetTy(
     }
 }
 
-fn semaStructFields(
-    mod: *Module,
-    arena: Allocator,
-    struct_type: InternPool.Key.StructType,
-) CompileError!void {
-    const gpa = mod.gpa;
-    const ip = &mod.intern_pool;
-    const decl_index = struct_type.decl.unwrap() orelse return;
-    const decl = mod.declPtr(decl_index);
-    const namespace_index = struct_type.namespace.unwrap() orelse decl.src_namespace;
-    const zir = mod.namespacePtr(namespace_index).file_scope.zir;
-    const zir_index = struct_type.zir_index;
+fn semaStructFieldsLen(zir: Zir, zir_index: Zir.Inst.Index) struct {
```

Comment:
Functions beginning `sema` generally begin semantic analysis of something (e.g. `semaDecl` analyzes a `Decl`, `semaStructFields` analyzes the field types of a struct). This should be something like `structZirInfo` instead.

Reply:
Renamed.